melosys-eessi-app/src/main/java/no/nav/melosys/eessi/metrikker/DLQMetrikkerCache.java

```diff
@@ -25,6 +25,7 @@ public class DLQMetrikkerCache {
     public void oppfriskDLQTypeOgAntall() {
         log.debug("Oppfrisker antall DLQ per type");
         List<KafkaDLQAntall> kafkaDLQTypeOgAntall = kafkaDLQRepositoy.countDLQByQueueType();
+        queueSizeMap.clear();
```

Comment:
Trenger vi det?

Reply:
Spurte ChatGPT om det samme skjønner du:

```
Will that overwrite if the value changes, or will there be multiple values for the same queuetype?
```


```
The queueSizeMap.put() function will overwrite the previous value associated with the given key (in your case, the queueType). The Map data structure in Java does not allow duplicate keys. Each key can map to exactly one value.

So, if you call queueSizeMap.put() with a queueType that's already in the map, the new value will replace the old value, not add a new entry.

Thus, you will always have the latest count values in your queueSizeMap for each queueType, which will be refreshed every 10 seconds due to the @Scheduled(fixedRate = 10000) annotation.

```

Reply:
Det trenges ikke her, det stemmer. Siden nye verdier hentes for alle køtyper. Skal fjerne